charts/karpenter/values.yaml

```diff
@@ -29,6 +29,12 @@ serviceMonitor:
   additionalLabels: {}
   # -- Endpoint configuration for the ServiceMonitor.
   endpointConfig: {}
+grafana:
```

Comment:
Do we expect to have other grafana-style configurations that we should put in this stanza or do we just collapse it down like we did to the serviceMonitor to `grafanaDashboards`? I'd be interested to see prior-art for other helm charts that also deploy Grafana dashboards. Maybe @stevehipwell has some insight here too?

Reply:
I think in the context of Helm you want to restrict the actual dashboard implementation to be as closely scoped as possible; think hidden variables for things such as namespace which are dynamic for Helm but fixed once Karpenter has been deployed. I'm not sure if any of these require additional "Grafana" configuration but I'd be tempted to use `grafana.dashboards` in case there ever was.

Reply:
I can do either way, I just started it so we can have this discussion. please let me know what you both decide on